### ISSUE TITLE
Batch_size larger than one drops the accuracy 

### ISSUE THREAD

**Issue by monaj07** [meetshah1995/pytorch-semseg#34]
@meetshah1995 @ibadami Thanks for making these codes publicly available.
Have you tried training with a `batch_size` larger than one? Why does it degrade the accuracy? I wonder if the cross_entropy2D messes up the batch orderings.

**Comment by meetshah1995** [comment]
Yes, I trained FCN on VOC and FRNN on Cityscapes on batch_size of >1 (8 and 4 resp), worked just fine.
